### PR TITLE
Update HPA Nucleus Segmentation (DPNUnet)

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.6200999.
The following version(s) will be added:
  - 6224243
  - 6221443

You can find their corresponding RDF files here:
  - https://zenodo.org/api/files/9f580ca7-5f85-41ac-be2b-201172ed09b3/rdf.yaml
  - https://zenodo.org/api/files/cbeebc4a-df16-4013-bc52-98c1b843baa4/rdf.yaml

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the BioImage.IO Spec validator (static validation);
- [ ] Passed the BioImage.IO CI tests (dynamic validations);
- [ ] The meta information for the RDF item is complete;
  - [ ] The tags is complete and describes the model;
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D);
  - [ ] Authors are provided;
  - [ ] Documentation is complete;
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: none specified

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: https://github.com/bioimage-io/collection-bioimage-io/pull/296

### CODE REVIEW

File: collection/10.5281/zenodo.6200999/resource.yaml [hunk 1]
```diff
@@ -12,17 +12,24 @@ versions:
...
   created: '2022-02-21 10:58:25.147491'
   status: accepted
   name: HPA Nuclei Segmentation (DPNUnet)
   version_name: revision 4
   rdf_source: https://zenodo.org/api/files/e5e8cef4-9d11-4eb1-8403-6d04859ae2f0/rdf.yaml
+- created: '2022-02-21 10:58:25.147491'
+  doi: 10.5281/zenodo.6201000
+  name: HPA Nuclei Segmentation (DPNUnet)
+  status: blocked
+  version_id: 10.5281/zenodo.6201000
```
Comment (oeway) [inline collection/10.5281/zenodo.6200999/resource.yaml:33]: @FynnBe Could you please take a look here, it seems there is some issue with the version_id.

File: collection/10.5281/zenodo.6200999/resource.yaml [hunk 2]
```diff
@@ -12,17 +12,24 @@ versions:
   version_id: 10.5281/zenodo.6221443
```
Comment (FynnBe) [inline collection/10.5281/zenodo.6200999/resource.yaml:12]:
```suggestion
  version_id: '6221443'
```

**Comment by FynnBe** [comment]
hmm.. I think we have to update all old versions ids like I did here now.

**Comment by FynnBe** [comment]
... yes I had. This PR seems to have been opened just before I merged my fork, so I overlooked to update it here.

**Comment by oeway** [comment]
> ... yes I had. This PR seems to have been opened just before I merged my fork, so I overlooked to update it here.

Ok, great, so we need to manually edit it? Does this explains why it fails the dynamic checks?

**Comment by FynnBe** [comment]
> Ok, great, so we need to manually edit it?

yes, but I did that already

> Does this explains why it fails the dynamic checks?

no, that is a different problem for the new version, as mentioned [here](https://github.com/bioimage-io/collection-bioimage-io/pull/306#discussion_r817186862) it looks like a file `test_input` or `test_output` might be a .tif file instead of a .npy file. Or the files are invalid .npy files in another way.

**Comment by oeway** [comment]
> no, that is a different problem for the new version, as mentioned [here](https://github.com/bioimage-io/collection-bioimage-io/pull/306#discussion_r817186862) it looks like a file `test_input` or `test_output` might be a .tif file instead of a .npy file. Or the files are invalid .npy files in another way.

This is wired then, I just merged the same one which passes the dynamic test: https://github.com/bioimage-io/collection-bioimage-io/pull/296

**Comment by constantinpape** [comment]
This is the same issue as in #301.
See https://zenodo.org/record/6221443#.YisHnTwo8UE - most of the files are missing.
Is there maybe something wrong with the uploader?

**Comment by oeway** [comment]
> This is the same issue as in #301. See > https://zenodo.org/record/6221443#.YisHnTwo8UE - most of the files are missing. Is there maybe something wrong with the uploader?

I figured that this is not the latest version, what happens is that I changed directly in Zenodo.

The latest version is this: https://zenodo.org/record/6224243

I will block this one now.

**Pull request merged**